packages/firestore/src/local/indexeddb_persistence.ts

```diff
@@ -489,7 +488,12 @@ export class IndexedDbPersistence implements Persistence {
             ).next(() => inactive);
           });
         }
-      );
+      ).catch(() => {
+        // Ignore primary lease violations or any other type of error.
```

Comment:
I think it's worth adding why it's OK to ignore all the errors here. 

Something along the line where you put in PR description: `There is no special recovery logic needed as the next primary will run the periodic GC.`

Reply:
Done.